From 78cb6d44c0852b6ac6ebc0b8a844fc278c762d7c Mon Sep 17 00:00:00 2001
From: Srinivas Pandruvada <srinivas.pandruvada@linux.intel.com>
Date: Fri, 28 Mar 2014 15:57:18 -0700
Subject: [PATCH 2/5] Remove rpmlint warning for manual page

Fix error:
manual-page-warning /usr/share/man/man5/thermal-conf.xml.5.gz 69: warning: numeric expression expected (got `c')
---
 man/thermal-conf.xml.5 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/man/thermal-conf.xml.5 b/man/thermal-conf.xml.5
index 7a77ba8..b59d86c 100644
--- a/man/thermal-conf.xml.5
+++ b/man/thermal-conf.xml.5
@@ -66,7 +66,8 @@ The configuration file format confirms to XML specifications. A set of tags defi
 	<!--UUID is optional, if present this will be matched. Both product name and UUID can contain wild card "*", which matches any platform.-->
 .TP
 	<UUID>Example UUID</UUID>
-.TP configuration file format confirms to XML specifications. A set of tags defined to define platform, sensors, zones,  cooling
+.TP
+	configuration file format confirms to XML specifications. A set of tags defined to define platform, sensors, zones,  cooling
        devices and trip points.
 
        <ThermalConfiguration>
-- 
2.0.0.rc0

